Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml

```diff
@@ -159,6 +167,22 @@
   - type: AccessReader
     access: [["Research"], ["Medical"]]
 
+- type: entity
+  parent: AirlockResearch
+  id: AirlockResearchLocked
+  suffix: Research, Locked
+  components:
+  - type: AccessReader
+    access: [["Research"]]
+
+- type: entity
+  parent: AirlockResearch
+  id: AirlockMedicalResearchLocked
+  suffix: Medical/Research, Locked
+  components:
+  - type: AccessReader
+    access: [["Research"], ["Medical"]]
+
```

Comment:
i think if we're going to add the darker purple research airlocks they should really just be sprite changes on the old sci airlocks, I don't see any reason to have both